api/v3/PaymentProcessor.php

```diff
@@ -170,10 +176,11 @@ function _civicrm_api3_payment_processor_pay_spec(&$params) {
  * @throws CiviCRM_API3_Exception
  */
 function civicrm_api3_payment_processor_refund($params) {
+  /** @var \CRM_Core_Payment $processor */
   $processor = Civi\Payment\System::singleton()->getById($params['payment_processor_id']);
   $processor->setPaymentProcessor(civicrm_api3('PaymentProcessor', 'getsingle', array('id' => $params['payment_processor_id'])));
   if (!$processor->supportsRefund()) {
-    throw API_Exception('Payment Processor does not support refund');
+    throw new CiviCRM_API3_Exception('Payment Processor does not support refund');
```

Comment:
Don't we throw API_Exception within apis & the wrapper throws the other?

Reply:
It would appear so!